### PR TITLE
Javadoc fix

### LINKED ISSUE #658: published API Javadoc jar is empty
The javadoc jar published at https://frcmaven.wpi.edu/artifactory/release/edu/wpi/first/shuffleboard/api/2020.3.2/ is empty

### PR DESCRIPTION
**Pull request by carbotaniuman**
Fix javadocs. This adds the right gradle option and fixes all of the doc comments. Depends on #673. Fixes #658.

### CODE REVIEW

File: plugins/networktables/src/main/java/edu/wpi/first/shuffleboard/plugin/networktables/sources/NetworkTableSource.java [hunk 1]
```diff
@@ -22,7 +22,7 @@
  * @implNote Subclasses must call {@link #setTableListener(TableListener) setTableListener()} after
  *           the super constructor call. If a subclass needs to implement {@link #close()}, its
  *           implementation <i>must</i> call {@code super.close()} to properly remove the listener
- *           from network tables.
+ *           from network tables.z`
```
Comment (AustinShalit) [inline plugins/networktables/src/main/java/edu/wpi/first/shuffleboard/plugin/networktables/sources/NetworkTableSource.java:25]: What is this?

File: api/src/main/java/edu/wpi/first/shuffleboard/api/properties/SavePropertyFrom.java [hunk 2]
```diff
@@ -36,31 +37,33 @@
...
  * class MyWidget implements Widget {
  *
- *  @literal @SavePropertyFrom(propertyName = "value", savedName = "speed")
- *  @literal @SavePropertyFrom(propertyName = "min", savedName = "minSpeed")
- *  @literal @SavePropertyFrom(propertyName = "max", savedName = "maxSpeed")
- *   private final Slider speedSlider = new Slider();
- *
+ *  {@literal @}SavePropertyFrom(propertyName = "value", savedName = "speed")
+ *  {@literal @}SavePropertyFrom(propertyName = "min", savedName = "minSpeed")
+ *  {@literal @}SavePropertyFrom(propertyName = "max", savedName = "maxSpeed")
```
Comment (AustinShalit) [inline api/src/main/java/edu/wpi/first/shuffleboard/api/properties/SavePropertyFrom.java:62]: Is this correct?

Comment (carbotaniuman) [reply api/src/main/java/edu/wpi/first/shuffleboard/api/properties/SavePropertyFrom.java:62]: Yes, I've checked.

File: plugins/networktables/src/main/java/edu/wpi/first/shuffleboard/plugin/networktables/sources/NetworkTableSource.java (hunk 1 above)
Comment (carbotaniuman) [reply plugins/networktables/src/main/java/edu/wpi/first/shuffleboard/plugin/networktables/sources/NetworkTableSource.java]: Removed.

**Comment by AustinShalit** [comment]
Why do you specify the java doc task options twice?

**Comment by carbotaniuman** [comment]
Do I just need to do it in `subprojects`?

**Comment by AustinShalit** [comment]
My understanding is that you only need it once in all projects but I could be wrong.

**Pull request closed without merging**